src/base_classes/forte_options.h

```diff
@@ -187,6 +194,55 @@ class ForteOptions {
      */
     std::vector<double> get_double_vec(const std::string& label);
 
+    /**
+     * @brief Set a boolean option
+     * @param label Option label
+     * @param val Option value
+     */
+    void set_bool(const std::string& label, bool val);
+
+    /**
+     * @brief Get a integer option
```

Comment:
The briefs should be Set here and after.

Reply:
Thanks for catching this!